models.py

```diff
@@ -30,6 +30,9 @@ def is_active(self):
     def mark_activity(self):
         self.updated_at = datetime.utcnow()
 
+    def mark_inactive(self):
+        self.updated_at = None
```

Comment:
Marking inactivity should not be related with updated_at. Other parts depends on this field.